package.json

```diff
@@ -81,7 +81,7 @@
     "publish": true
   },
   "engines": {
-    "node": "^14.17.0 || ^16.13.0 || >=18.0.0"
```

Comment:
Will this mean we have to bump engines in all npm subdependencies before we can move to this version?

Reply:
No, we only run `template-oss` in the latest Node version supported by the package. So currently everything runs it in Node 20, but I kept Node 18 support just because.